doc/ironic-guide.rst

```diff
@@ -51,6 +51,27 @@ Set ironic_console_serial_speed in ``/etc/kolla/globals.yml``:
 
 .. _web_console_documentation: https://docs.openstack.org/ironic/latest/admin/console.html#node-web-console
 
+Booting with iPXE
+=================
+To enable iPXE support, set enable_ironic_ipxe in ``/etc/kolla/globals.yml``:
+
+::
+
+    enable_ironic_ipxe: "yes"
+
+This will enable deployment of a docker container called ironic_ipxe, running
+the web server which iPXE uses to obtain it's boot images.
+
+The port used for the iPXE webserver is controlled via ironic_ipxe_port in
+``/etc/kolla/globals.yml``:
+
+::
+
+    ironic_ipxe_port: "8089"
+
+Ironic will configure the DHCP settings to enable chainloading iPXE from an
```

Comment:
So just wondering, can you set the BIOS to iPXE mode only, and will this automatically skip the chain loading stuff?

Reply:
Ah - I think I see. This is handled in the dnsmasq rules?

Reply:
I thought you normally had to burn iPXE to the bootrom of the network card (https://ipxe.org/howto/romburning), but if someone bundles it, yeah, it should work no problem.

Reply:
Yeah, dnsmasq for ironic-inspector, but ironic configures neutron for normal deployment:

https://github.com/openstack/ironic/blob/ca0cff144a4b621a9c7afb891017043e80ecdc50/ironic/common/pxe_utils.py#L333